### PR TITLE
complete the quick guide example

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
JS/TS changes look good. Can you please also fix the paratheses alignment in Python and ReactNative examples.

https://amplitude.slack.com/archives/C02E1H87E2C/p1663163693714989?thread_ts=1663106803.691659&cid=C02E1H87E2C

image

```
amplitude.track(BaseEvent(
          event_type="type of event",
          user_id="USER_ID",
          device_id="DEVICE_ID",
          event_properties={
                "source": "notification"
          }
))
```

or 

```
amplitude.track(
       BaseEvent(
          event_type="type of event",
          user_id="USER_ID",
          device_id="DEVICE_ID",
          event_properties={
                "source": "notification"
          }
      )
)
```

File: docs/data/sdks/index.md [hunk 1]
```diff
@@ -193,7 +265,20 @@ Use this guide to get started with the Amplitude SDKs. Choose your target langua
...
         ```
 
-        Learn more about [Ampli Node](../sdks/typescript-node/ampli/).
+    === "JavaScript"
+
+        ```ts
+
+        ampli.load();
+
+        ampli.yourEventType("ampli-user-id", {
```
Comment (liuyang1520) [inline docs/data/sdks/index.md:274]: Minor, double quote -> single quote?

File: docs/data/sdks/index.md [hunk 2]
```diff
@@ -793,6 +951,19 @@ Use this guide to get started with the Amplitude SDKs. Choose your target langua
         ```ts
 
         ampli.load();
+
+        ampli.yourEventType({
+            stringProp: "Strongly typed property",
```
Comment (liuyang1520) [inline docs/data/sdks/index.md:956]: Ditto?

**Review by liuyang1520 — APPROVED** [review]

**Comment by caseyamp** [comment]
@yuhao900914 are we waiting on anything here?

File: docs/data/sdks/index.md [hunk 3]
```diff
@@ -556,71 +660,102 @@ Use this guide to get started with the Amplitude SDKs. Choose your target langua
...
-    );
+        ```
+
+    === "Kotlin"
+
+        ```kotlin
+        Ampli.getInstance().load()
+
+        Ampli.getInstance().yourEventType("ampli-user-id",
+            YourEventType(
```
Comment (justin-fiedler) [inline docs/data/sdks/index.md:714]: nit: align parentheses

**Comment by yuhao900914** [comment]
> @yuhao900914 are we waiting on anything here?

**Review by justin-fiedler — CHANGES_REQUESTED** [review]
1. missing tab in RN TS example:
image
2. JRE Java 
image
3. Browser standardize to single quote:
image

**Pull request closed without merging**